src/Dapr.PluggableComponents.AspNetCore/WebApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Dapr.PluggableComponents;
+
+public static class WebApplicationBuilderExtensions
+{
+    public static string AddDaprPluggableComponentsServices(this WebApplicationBuilder builder, DaprPluggableComponentsApplicationOptions options)
+    {
+        string socketExtension = options.SocketExtension
+            ?? Environment.GetEnvironmentVariable(Constants.EnvironmentVariables.DaprComponentsSocketsExtension)
+            ?? Constants.Defaults.DaprComponentsSocketsExtension;
+
+        // TODO: Add support for native (i.e. non-WSL) Windows.
```

Comment:
Windows 10 and above support UDS (and I expect .NET 6 on Windows as well).  I'd need to do further testing to see whether it actually works the way Dapr expects.  (The Dapr runtime doesn't seem to have a Windows-compatible default for the socket folder, so there'd likely to be changes needed there.)  I did some early testing and was able to get the gRPC service listening on the socket, but couldn't connect (via Postman) in the same way I could on Mac.

Reply:
I think it will be more a matter of ensuring Go has support for Windows UDS. If Go lacks this support, `daprd` lacks this support and there is no point adding this to the plugins.

Reply:
I did a little investigating this afternoon; with tweaks to `daprd` to make it attempt to connect on Windows, I get as far as trying to open the connection before it gets stuck--it doesn't return an error, just doesn't connect.  I *was* able to make an initial connection to the service using other tools (like `curl` and `grpcurl`), which would seem to indicate that the socket is active.  I'm willing to live with it not being a supported scenario initially.